openmls/src/group/core_group/mod.rs

```diff
@@ -189,6 +190,11 @@ impl CoreGroupBuilder {
             .with_required_capabilities(required_capabilities);
         self
     }
+    /// Set the [`Capabilities`] of the [`CoreGroup`].
```

Comment:
This confused me for a second because this sounds like the capabilities are a property of the CoreGroup and not the LeafNode. Maybe "Capabilities of the group creator's LeafNode" or something like that would be more accurate?